src/main/kotlin/io/hsar/wh40k/combatsimulator/model/UnitInstance.kt

```diff
@@ -13,7 +13,7 @@ import io.hsar.wh40k.combatsimulator.model.unit.Unit
  * This class may contain dynamic information that changes as combat progresses.
  */
 class UnitInstance(
-        val name: String,
+        val name: String,  // #TODO how is this different from the name member variable of unit?
```

Comment:
I added this so that you can have "bandit1" and "bandit2" which are both "bandit". Maybe we should rename `UnitInstance` and `Unit` to be more useful.

Reply:
Nah that makes sense now you've explained it.